src/devices/Pca9685/samples/Pca9685.Sample.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Pca9685;
+using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+
+namespace Iot.Device.Pca9685.Samples
+{
+    /// <summary>
+    /// This sample is for Raspberry Pi Model 3B+
+    /// </summary>
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            var busId = 1;  // /dev/i2c-1
+            var deviceAddress = 0x40;
```

Comment:
As far as I know, this board I2C address is customizable. Can we add a class to help calculate the address? Of course, it's not necessary. I can do it after the code merge.

Reply:
@ZhangGaoxing is it customizable with external pins? If so I'd recommend adding const for whatever the default value is and put that info in the README instead (not necessary to go into details just mention that and point to the datasheet).

Reply:
The universal address of the board is 0x70. Change the address by changing the hardware address pin on the board. Custom address calculate method: **address = 0x40 | 0b_A5A4_A3A2A1A0**
@HumJ0218 @krwq 
![42022263-6e072926-7aef-11e8-8193-c27a6e64700b](https://user-images.githubusercontent.com/15957347/53147159-a4c81300-35e1-11e9-9f1b-aa51189836bb.png)

Reply:
I'd with just a const `I2cAddress`/`DefaultI2cAddress` and you feel like it perhaps also add an enum `CustomAddressSetting` with `[Flags]` with `A0`-`A5` values and a static function `GetCustomAddress(CustomAddressSetting addressSetting)`